pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -190,34 +200,46 @@
             "type": "string"
           },
           "type": "object",
+          "description": "defines the annotations to apply to the service account",
```

Comment:
also kind of noisy IMO since the key is `serviceaccounts.annotations`, same with `name`, `namespace`, etc.